wecode-live/pyramid
# Add pyramid code for Python and JS

### What does this PR do?
It adds code for full pyramid in python and half pyramid in JavaScript
### Description of Tasks to be completed?
- Add code for full pyramid in python in file ```pyramid.py```
- Add code for half pyramid in JavaScript in file ```pyramid.js```
### How should this be tested?
- After cloning to local machine, cd into the python directory and run ```python pyramid.py``` to see full pyramid in console.
- cd into javascript directory and run ```node pyramid.js``` to see half pyramid in console.
### Screenshots
![pyramid](https://user-images.githubusercontent.com/24798364/31225046-2d10a3aa-a9c8-11e7-906e-376810fdc42f.PNG)

![pyramid2](https://user-images.githubusercontent.com/24798364/31225063-3b4251bc-a9c8-11e7-8b76-eb3b04a7731b.PNG)
